src/action.cr

```diff
@@ -147,4 +172,8 @@ class Action
 
     @config["shell_expansion"].as_bool
   end
+
+  private def perform_param_expansion? : Bool
+    @config.fetch("param_expansion", YAML.parse("false")).as_bool.as(Bool)
```

Comment:
Do we need that `.as(Bool)` on the end?

Reply:
Hmmm, not sure, tbh. I feel like I maybe had to put this here because `cr` had trouble resolving the type? I'll test it again and get back to you.